hack/util.sh

```diff
@@ -641,7 +641,11 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template="{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+	local config="--config=${ADMIN_KUBECONFIG}"
+	local tmplsubsets='{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}'
+	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template='"'$tmplsubsets' $config"' || echo 0)" != "0" ]]' $((5*TIME_MIN))
```

Comment:
Very interesting. There's a bigger refactor that will fix this that's currently blocked on me focusing on the e2e/conformance wedges... hopefully that will be in next week and at that point I can refactor this code to use it.